extensions/amp-wordpress-embed/1.0/base-element.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PreactBaseElement} from '#preact/base-element';
+import {WordpressEmbed} from './component';
```

Comment:
Also per `capital_P_dangit`, although perhaps this casing is required by AMP's style guide.

```suggestion
import {WordPressEmbed} from './component';
```